src/main/java/uk/gov/hmcts/reform/roleassignment/feignclients/configuration/DataStoreApiFallback.java

```diff
@@ -16,9 +16,24 @@ public String getServiceStatus() {
 
     @Override
     public Case getCaseDataV2(String caseId) {
-        return Case.builder().id(caseId)
+        Case dummyCase = null;
+
+        if (caseId.equals("1234567890123456")) {
```

Comment:
it would be neat and clean if you use switch operation here.

Reply:
Yes, I will update it.